### ISSUE TITLE
On course page allow linking to previous iterations of courses

### ISSUE THREAD

**Issue by BHenkemans** [GEWIS/gewisweb#1726]
### What would you like?

I would like to have the previous iterations (or future iterations) of course to be added to the course page. Something like this.

![image](https://github.com/GEWIS/gewisweb/assets/38168308/c970cb35-1d8a-47fc-b891-2df785226280)

### Why is this needed?

Due to the bachelor college redesign, a lot of courses have changed slightly but not enough to make previous documents irrelevant. Therefore, informing students of the previous iterations of the course could be useful

### Other information

It would require the Educational Officer link the courses, but he himself was interested in this feature too, so I am sure he wont mind too much (@JortvD ;))

**Comment by JortvD** [comment]
Yeah I had a very similar idea for the similar courses. My way to do it would be to create a many-to-many table of course ID to course ID (and joined with the course info), where one course points to other courses that are similar. That way you can also have some courses point to another course but not point back. Then in the admin panel the easiest (MVP) solution would be to have single text field with comma seperated course IDs and a check on the form if the courses exist and if the mappings are unique.

**Comment by tomudding** [comment]
Added through GH-1727.
